### DISCUSSION TITLE
vod files questions

### DISCUSSION THREAD

**Discussion by 111100001** [MrBrax/LiveStreamDVR#522] [General]
im trying to write a program that checks the directories that LiveStreamDVR downloads vods in, and then once it sees the DVR has finished downloading/capturing the stream, it uploads it to the internetarchives.

my question is i need to know if you use a lock file or another specific marker while proccessing/downloadin the file so i can use that in my program to indicate that LiveStreamDVR is done downloading/capturing the stream

**Comment by mr-flibble** [comment]
I couldn't find a way, so my workround for this was simple wait

`find ./ -cmin +1440 -print0 |`

**Comment by 111100001** [comment]
thats probably what im gonna do. thank you v much

**Comment by MrBrax** [comment]
vod json files contain is_capturing, is_finalized etc

**Reply by 111100001** [reply]
nicce! thank you
